NEWS.md

```diff
@@ -1,5 +1,9 @@
 # EpiNow2 (development version)
 
+## Package
+
+- The `future` and `future.apply` packages are now installed in the `R CMD CHECK` workflow to ensure that the `test-coverage` workflow runs without errors. By @jamesmbaazam in #827 and reviewed by @<REVIEWER>.
+
```

Comment:
I would normally not add a news item for infrastructure changes as this information is irrelevant to the user - unless you think it hides credit (in which case we could keep it but not under a "Package" heading I think).